trigger-actions-framework/main/default/classes/TriggerActionFlowBypassProcessor.cls

```diff
@@ -0,0 +1,33 @@
+public inherited sharing abstract class TriggerActionFlowBypassProcessor {
```

Comment:
We need to add the Apache license header for this class.

I _think_ that the header needs to include Google LLC as the copyright holder. I can check with our internal OSS team to confirm.

Reply:
no need to confirm - I think it makes sense for all of the files to have the license at the top purely to be consistent